roachtest: fix cdc/kafka-chaos-multiple-sink-nodes

andyyang890 opened the pull request:
This patch fixes `cdc/kafka-chaos-multiple-sink-nodes` by swapping
the order of the sink and workload nodes since the roachprod prometheus
code assumes the last node is the workload node.

Fixes #127110

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/133833)
<!-- Reviewable:end -->

andyyang890 committed:
roachtest: fix cdc/kafka-chaos-multiple-sink-nodes

This patch fixes `cdc/kafka-chaos-multiple-sink-nodes` by swapping
the order of the sink and workload nodes since the roachprod prometheus
code assumes the last node is the workload node.

Release note: None

andyyang890 commented:
TFTRs!

bors r=DarrylWong,asg0451
